src/lib/appVersion.js

```diff
@@ -0,0 +1,4 @@
+import pkg from "../../package.json";
```

Comment:
this file is not used right now - consider getting the version from this file import in `App.tsx` or removing this file entirely